llmfoundry/data/packing.py

```diff
@@ -393,11 +357,11 @@ def split_big_batch(raw_batch_size: int) -> List:
                 batches[idx].update({key: split})
         return batches
 
-    def profile_packing(raw_batch_size: int) -> Tuple[float, float]:
-        packer = BinPackWrapper(
+    def profile(raw_batch_size: int) -> Tuple[float, float]:
+        packer = BinPackCollator(
             collator=lambda x: x,
```

Comment:
@alextrott16 Do you think we should be passing in dataloader.collate_fn here instead of `lambda x: x` ?

Reply:
I don't think we need to, unless you prefer to refactor the surrounding code. The dataloader that is set up in this script already applies the seq2seq collator function, so, in the context of this script, having `BinPackCollator` wrap an identify function should be correct in the context of the current logic.